### DISCUSSION TITLE
Episode 127: Flux, ChatGPT & More with Tristan Brindle

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#18] [Episode]
Episode Link: https://adspthepodcast.com/2023/04/28/Episode-127.html

Feel free to write comments or ask questions 🙂

**Comment by tothambrus11** [comment]
Bryce, I think they are already working on the PPT copilot :D 
https://www.microsoft.com/en-us/microsoft-365/blog/2023/03/16/introducing-microsoft-365-copilot-a-whole-new-way-to-work/
